### PR TITLE
oneapi-cpi : rename dependency for oneDNN folder name change

### PR DESCRIPTION
**Pull request by louie-tsai**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

### CODE REVIEW

File: AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
  "categories": ["Toolkit/oneAPI AI And Analytics/AI Getting Started Samples"],
  "description": "This sample illustrates how to run ILIT to quantize the FP32 model trained by Keras on Tensorflow to INT8 model to speed up the inference.",
  "languages": [{"python":{}}],
- "dependencies": ["tensoflow","notebook","matplotlib"],
  "os": ["linux"],
```
Comment (anjgola) [inline AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json:7]: This should be 'tensorflow' and 'iLiT'. I was just creating a PR for this but can you please edit this one.

**Review by anjgola — DISMISSED** [review]

File: AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json (hunk 1 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json:7]: ok. got it.

Comment (anjgola) [reply AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json:8]: Sorry to ping again but the letter 'r' is missing from tensorflow word.

Comment (anjgola) [reply AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json:8]: MErging this, will fix the r directly.

**Review by anjgola — APPROVED** [review]

File: AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json (hunk 1 above)
Comment (anjgola) [reply AI-and-Analytics/Getting-Started-Samples/iLiT-Sample-for-Tensorflow/sample.json:8]: r is fixed.

**Pull request merged**